app/routes/quotes/QuotesRoute.js

```diff
@@ -22,14 +22,17 @@ const qsParamsParser = (search) => ({
 });
 
 const mapStateToProps = (state, props) => {
+  const { approved } = qsParamsParser(props.location.search);
   const { pagination } = selectPaginationNext({
     endpoint: `/quotes/`,
-    query: qsParamsParser(props.location.search),
+    query: { approved },
```

Comment:
Does this really have to be a variable? 🤔 I don't seem to understand how this changes anything

Reply:
No it does not have to be a variable, its a leftover from when it was passed to the selector